docs/getting-started/getting-started.md

```diff
@@ -115,3 +115,22 @@ To stop your application project, go to the Liberty tools dashboard, right click
 ![Stop action](images/maven-devModeStopped.png)
 
 Once the project is stopped, the terminal in which it ran is closed.
+
+## Configuring a Liberty server (editing server.xml)
+
+1. Start the project in dev mode, using the dashboard **Start** command above.  This will install the Liberty features required for your app and allow the generation of a corresponding server.xml XSD schema file.
+
+2. Open your server.xml file.  This should be opened in the "Generic Text Editor" since the function here is tied to specific editors.
```

Comment:
but it should not take an overt action on the part of a user to open it in the Generic editor - double clicking the file will automatically open in the Generic editor.  So maybe: 

2. Double-click on your server.xml file which will cause it to be opened in the "Generic Text Editor" since the function here is tied to specific editors.

?

Reply:
Good suggestion.